client/components/SingleModel.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { connect } from 'react-redux';
+
+class SingleModel extends React.Component{
+    constructor(){
+        super()
+        this.state = {
+            model:'',
+            description:'',
+            price:0,
+            image:''
+        }
+    }
+    componentDidMount(){
+        const model = this.props.products.find(e => e.id === this.props.match.params.id*1)
+        this.setState({
+            model: model.model,
+            description: model.description,
+            price: model.price,
+            image: model.image
+        })
+    }
+    render(){
+        console.log(this.state)
+        const {model,description,price,image} = this.state
```

Comment:
Great use of destructuring here!

Reply:
Though I would say inconsistent linting, if you are using spacing on import destructures, use it on object destructures.

e.g.

```javascript
const { a, b, c } = thing;
```

Reply:
Great catch - as an extension of this it might be worth talking about normalizing your teams linters if you haven't already. I'd think that most teams consider it best practice to make sure that your linter is run before code is merged in, my team at work runs the linter as part of our CI process and blocks merges until fixes are made.

Reply:
Half our team doesn't use linters, so they would either have to install one or manually perform this task. lol

Reply:
You can install linters as part of your projects dependencies instead of globally. Then you can implement CI to enforce it (Travis). Theres nothing manual here, just a missing config, some installations, and a lack of CI.